ts/features/bonus/bpd/components/transactionItem/BpdTransactionItem.tsx

```diff
@@ -24,11 +24,19 @@ type Props = {
  * TODO: move the get subtitle in the combiner and remove this component?
  * @param transaction
  */
-const getSubtitle = (transaction: BpdTransaction) =>
-  `${localeDateFormat(
-    transaction.trxDate,
-    I18n.t("global.dateFormats.dayMonthWithTime")
-  )} · € ${formatNumberAmount(transaction.amount)} `;
+export const getSubtitle = (transaction: BpdTransaction) => {
+  const isMidNight =
+    transaction.trxDate.getHours() + transaction.trxDate.getMinutes() === 0;
```

Comment:
What you think about adding also the seconds to the calculation?
```suggestion
    transaction.trxDate.getHours() + transaction.trxDate.getMinutes() + transaction.trxDate.getSeconds() === 0;
```
In this way we can render the time for a transaction with timestamp `2020-12-13T00:00:31+01:00` that is a valid transaction.